common/tools/dev-tool/src/commands/run/testNodeTSInput.ts

```diff
@@ -25,7 +25,7 @@ export default leafCommand(commandInfo, async (options) => {
   const reporterArgs =
     "--reporter ../../../common/tools/mocha-multi-reporter.js --reporter-option output=test-results.xml";
   const defaultMochaArgs = `${
-    isModuleProj ? "" : "-r esm "
+    isModuleProj ? "--loader=ts-node/esm " : "-r esm "
```

Comment:
In the inference library, we have this in a [config file](https://github.com/Azure/azure-sdk-for-js/blob/522bc425ba6728e185a5d09578b033fe8659f752/sdk/openai/openai/.mocharc.json#L4). @jeremymeng Is there any harm to move that option to the dev-tool command?

Reply:
No, I like centralized options in dev-tool which make it easier to apply changes for all affected packages.  We may want to update other existing modular .mocharc if any also uses this.